src/components/PostCard/PostTitle/PostTitle.js

```diff
@@ -1,13 +1,11 @@
 import React from 'react'
-import { decode } from 'ent'
 import Highlight from 'components/Highlight'
 import './PostTitle.scss'
 
 export default function PostTitle ({
   title,
   highlightProps
 }) {
-  title = decode(title)
```

Comment:
why remove this?

Reply:
I changed how that prop is pass - it's passed as a string now, not an HTML entity and no longer needs to be decoded.